data_diff/databases/duckdb.py

```diff
@@ -167,12 +167,16 @@ def select_table_schema(self, path: DbPath) -> str:
         database, schema, table = self._normalize_table_path(path)
 
         info_schema_path = ["information_schema", "columns"]
+
         if database:
             info_schema_path.insert(0, database)
+            dynamic_database_clause = f"'{database}'"
+        else:
+            dynamic_database_clause = "current_catalog()"
```

Comment:
This logic inherits the database already configured for the duckdb/motherduck connection:
`filepath = "md:data_diff_demo?motherduck_token=${MOTHERDUCK_TOKEN}" # `